src/components/forms/transaction/MainSplit.tsx

```diff
@@ -52,11 +54,11 @@ export default function MainSplit({
   }, [txCurrency, date, form.formState.isDirty, account, prices]);
 
   React.useEffect(() => {
-    if (form.formState.isDirty) {
+    if (form.formState.isDirty && splits.length > 1) {
```

Comment:
This is so we support split events for investments. I don't think there is another case where splits length is only 1 so using this condition for now.